MinimalCirclesPath.py

```diff
@@ -64,33 +64,30 @@ def getDistance(startLat, startLong, destLat, destLong):
 
 # pathWidth is the desired scaned distance from play diameter, radius is radius of circle passed to NOTAM
 def getPath( startLat, startLong, destLat, destLong, radius, pathWidth):
-
-    # TODO For actual application you would need to capture N/S/E/W
-
     
-      
-    # gets total distance from start to finish
-    totalDistance = getDistance(startLat, startLong, destLat, destLong)
-
     # gets step distance (pythagorean theorum)
     stepDistance = 2 * sqrt((radius)**2-(pathWidth/2)**2)
 
     # gets direction
     bearing = calculateBearing(startLat, startLong, destLat, destLong)
 
+    # updates the start so the area in the opposite direction of the path is right on the edge of the first circle
+    updatedStart = nextPoint(startLat,startLong, bearing, radius - (pathWidth/2) )
```

Comment:
I'm confused how the nextPoint function is going to return a point "in the opposite direction of the path" when you give it the same `bearing` as the path direction, and `radius - (pathWidth/2)` is seemingly always a positive value

Reply:
I see why that's confusing. A lot of these things are really hard to write out in words so I apologize for the confusion. I'm stepping in the direction of the destination a fraction of the radius so that a full radius from that new point encapsulates the start and some area before the start. Basically I am taking advantage of the fact that I will have a circle covering the distance radius in both directions. If I need to draw a picture to make it clearer I can

Reply:
I think I finally came around to understanding the comment as-it-is-written.  The assumption is that radius is always bigger than pathWidth/2, which I think certainly is true in all the unit tests and the code as it is written today.  However, having such an assumption in the code may be worth codifying as a check (and raise an Exception if violated) to ensure that future changes (e.g., making the radius and pathWidth something the user can specify or control) don't violate such an assumption.